arangodb-net-standard.Test/GraphApi/GraphApiClientTest.cs

```diff
@@ -185,5 +185,57 @@ public async Task GetGraphEdgeCollectionsAsync_ShouldThrow_WhenGraphIsNotFound()
             Assert.Equal(HttpStatusCode.NotFound, exception.ApiError.Code);
             Assert.Equal(1924, exception.ApiError.ErrorNum); // GRAPH_NOT_FOUND
         }
+
+        [Fact]
+        public async Task PostVertexCollectionAsync_ShouldSucceed()
+        {
+            // Create a new graph
+
+            string graphName = nameof(PostVertexCollectionAsync_ShouldSucceed);
+
+            PostGraphResponse createResponse = await _client.PostGraph(
+                new PostGraphBody()
+                {
+                    Name = graphName
+                });
+
+            // Add a vertex collection
+
+            string clxToAdd = nameof(PostVertexCollectionAsync_ShouldSucceed);
+
+            PostVertexCollectionResponse response = await _client.PostVertexCollectionAsync(
+                graphName,
+                new PostVertexCollectionBody()
+                {
+                    Collection = clxToAdd
+                });
+
+            Assert.Equal(HttpStatusCode.Accepted, response.Code);
+            Assert.False(response.Error);
+
+            PostVertexCollectionModifiedGraph graph = response.Graph;
+
+            Assert.Contains(clxToAdd, graph.OrphanCollections);
+        }
+
+        [Fact]
+        public async Task PostVertexCollectionAsync_ShouldThrow_WhenGraphIsNotFound()
+        {
+            string graphName = nameof(PostVertexCollectionAsync_ShouldThrow_WhenGraphIsNotFound);
+
+            var ex = await Assert.ThrowsAsync<ApiErrorException>(async () =>
+            {
+                await _client.PostVertexCollectionAsync(graphName, new PostVertexCollectionBody()
+                {
+                    Collection = "VertexCollectionThatShouldNotBeCreated"
+                });
+            });
+
+            ApiErrorResponse apiError = ex.ApiError;
+
+            Assert.True(apiError.Error);
+            Assert.Equal(HttpStatusCode.NotFound, apiError.Code);
+            Assert.Equal(1924, apiError.ErrorNum);
```

Comment:
Can you add a comment here for 1924 meaning?

Reply:
Ok I can do that but I don't think it adds much value. Here we care mainly about the fact that our data model carries the value over through the property, and less about what the value means.